docs/sources/whatsnew/whats-new-in-v11-5.md

```diff
@@ -0,0 +1,59 @@
+---
+description: Feature and improvement highlights for Grafana v11.5
+keywords:
+  - grafana
+  - new
+  - documentation
+  - '11.5'
+  - release notes
+labels:
+products:
+  - cloud
+  - enterprise
+  - oss
+title: What's new in Grafana v11.5
+posts:
+  - title: Cloud Migration Assistant
+    items:
+      - docs/grafana-cloud/whats-new/2025-01-10-grafana-cloud-migration-assistant-supports-all-plugins-and-grafana-alerting.md
+  - title: Dashboards and visualizations
+    items:
+      - docs/grafana-cloud/whats-new/2024-10-16-redesigned-ad-hoc-filters-for-dashboards.md
+      - docs/grafana-cloud/whats-new/2024-11-19-new-regular-expression-option-for-extract-fields-transformation.md
+      - docs/grafana-cloud/whats-new/2024-09-04-sharing-drawer.md
+      - docs/grafana-cloud/whats-new/2024-12-16-customizable-shareable-dashboard-panel-images.md
+  - title: Reporting
+    items:
+      - docs/grafana-cloud/whats-new/2024-10-21-theme-options-for-reporting.md
+      - docs/grafana-cloud/whats-new/2024-12-02-pdf-export-improvements-in-ga.md
+  - title: Alerting
+    items:
+      - docs/grafana-cloud/whats-new/2025-01-22-rbac-for-alerting-notifications.md
+      - docs/grafana-cloud/whats-new/2025-01-22-rbac-for-notification-policies.md      
+  - title: Data sources
+    items:
+      - docs/grafana-cloud/whats-new/2025-01-09-elasticsearch-cross-cluster-search-support.md
+      - docs/grafana-cloud/whats-new/2024-11-12-open-search-datasource-now-supports-private-datasource-connect.md
+      - docs/grafana-cloud/whats-new/2024-12-04-time-series-macro-support-in-visual-query-builder-for-sql-data-sources.md
+  - title: Authentication and authorization
+    items:
+      - docs/grafana-cloud/whats-new/2025-01-07-oauth-and-saml-session-handling-improvements.md
+  - title: Plugins
+    items:
+      - docs/grafana-cloud/whats-new/2025-01-10-plugin-frontend-sandbox.md
+  - title: Public dashboards
+    items: 
+      - docs/grafana-cloud/whats-new/2024-09-09-public-dashboards-are-now-shared-dashboards.md
+whats_new_grafana_version: 11.5
+weight: -47
+---
+
+# What’s new in Grafana v11.5
+
+Welcome to Grafana 11.5! Read on to learn about new sharing, reporting, and export options, cross-cluster search for Elasticsearch, PDC support for several new data sources, and more. The Grafana Cloud Migration Assistant is in public preview and now supports all plugins and Grafana Alerts, in addition to dashboards, folders, and data sources. We've also made it more secure to run third-party apps and data sources, and improved user session handling for OAuth and SAML.
```

Comment:
To fix the Vale linting

```suggestion
Welcome to Grafana 11.5!
Read on to learn about new sharing, reporting, and export options, cross-cluster search for Elasticsearch, PDC support for several new data sources, and more.
The Grafana Cloud Migration Assistant is in public preview and now supports all plugins and Grafana Alerts, in addition to dashboards, folders, and data sources.
We've also made it more secure to run third-party apps and data sources, and improved user session handling for OAuth 2.0 and SAML.
```

Reply:
We're linting for semantic line breaks?

Reply:
No sorry, that's just to make future diffs easier.

It's that it should be OAuth 2.0: https://developers.google.com/style/word-list#oauth-20.
Although Vale is already not complaining because OAuth is the correct spelling and Oauth isn't.